### ISSUE TITLE
GIT_REPOSITORY_URL logo

### ISSUE THREAD

**Issue by koeaw** [acdh-oeaw/apis-core-rdf#1331]
I'd like to suggest to change the logo used for `GIT_REPOSITORY_URL` from the [Git](https://git-scm.com/book/en/v2) logo (favicon for the official Git documentation, Git book) to one that reflects the platform that code is hosted on, in our case: the GitHub logo.

This reflects icon usage for code hosting platforms/linkage to repos across the web.

If we want to use a generic fallback, maybe we can find a logo that's not already associated with one very specific website, i.e. the one linked above, as that's confusing.

**Comment by koeaw** [comment]
The not-orange version of the Git logo would already be somewhat better than the orange one.

**Comment by koeaw** [comment]
Maybe we can find a variation on the theme of nodes/lines, which techies will recognise as representation of code/commits but which can't be confused with a particular tool or website. [Non-free example](https://www.alamy.com/git-branch-icon-line-isolated-on-white-background-black-flat-thin-icon-on-modern-outline-style-linear-symbol-and-editable-stroke-simple-and-pixel-p-image502916449.html).

**Comment by b1rger** [comment]
> to one that reflects the platform that code is hosted on

we can not know on which platform a project that uses `apis-core-rdf` is hosted on

> This reflects icon usage for code hosting platforms/linkage to repos across the web.

feel free to cite your sources

> If we want to use a generic fallback, maybe we can find a logo that's not already associated with one very specific website, i.e. the one linked above, as that's confusing.

the logo is the official git logo. its associated with the website you linked to, because thats the official git website.

**Comment by koeaw** [comment]
> we can not know on which platform a project that uses apis-core-rdf is hosted on

True. Neither can we know if such a project ultimately uses Git for version control.

The Material Symbol for [code](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:code:FILL@0;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%235f6368) could be used as more neutral alternative. The symbol for [folder data](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:folder_data:FILL@0;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%235f6368) would likely also be understood. `CODE_REPOSITORY`, `REPOSITORY_URL`  or simply `REPOSITORY` (wording used by [PyPI](https://packaging.python.org/en/latest/specifications/well-known-project-urls/#well-known-labels), [Poetry](https://python-poetry.org/docs/pyproject/#repository)) would make for less opinionated variable names.

**Comment by b1rger** [comment]
> > we can not know on which platform a project that uses apis-core-rdf is hosted on
> 
> True. Neither can we know if such a project ultimately uses Git for version control.

But we can be pretty sure about it (i.e. of all the Debian package sources roughly 63% of the upstream sources use git, the remaining 37% are distributed over 7 different version control systems). Also all acdh projects use git, both on github and on a self hosted gitlab.

**Comment by sennierer** [comment]
@koeaw if this is still an issue for you, can you please add a PR to change the Git logo for the [code](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:code:FILL@0;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%235f6368) you suggested. If not please just close the issue.
